test/fixtures/automation/actions.yml

```diff
@@ -22,3 +22,13 @@ four:
   automation_rule: four
   type: Automation::AddMessageThreadTagAction
   action_object: ssd_external_visible (Tag)
+
+five:
+  automation_rule: five
+  type: Automation::AddMessageThreadTagAction
+  action_object: ssd_attachment_matched (Tag)
+
+six:
+  automation_rule: five
+  type: Automation::ChangeMessageThreadTitleAction
+  value: "New title - ${title}"
```

Comment:
Ok, rozumiem tomu teda tak, ze do tohto ${title} sa vlastne vklada nazov vlakna a mozem si nastavit na urovni akcie, ze ako sa to premenuje. Asi good enough na teraz.

(Ako nejake rozsirenie si viem predstavit, ze budes matchovat regexp a vytiahnes z neho nieco co tu vlozis.)